travis/gen-index.sh

```diff
@@ -170,4 +170,12 @@ echo "</ul>" >> ${index_page}
 echo "</body>" >> ${index_page}
 echo "</html>" >> ${index_page}
 mv ${index_page} ${directory}
-echo ${name}.mate-desktop.dev > ${directory}/CNAME
+
+case ${name} in
+caja-actions|pluma-plugins)
+#   echo mate-desktop.github.io/${name} > ${directory}/CNAME
```

Comment:
No need to set an cname when mate-desktop.github.io is used.
CNAME file doesn't exists in this case.
Cname is only used for custom dns.

Reply:
done

Reply:
Let see how it works https://github.com/mate-desktop/caja-actions/pull/16